### ISSUE TITLE
[NEW] Многократный вызов функций ТекущаяДатаСеанса (ТекущаяДата) в рамках одного метода

### ISSUE THREAD

**Issue by artbear** [1c-syntax/bsl-language-server#1521]
## Описание проблемы, ошибки, которую надо диагностировать

Следует избегать в коде одной процедуры (функции) многократного обращения к функции `ТекущаяДатаСеанса (ТекущаяДата)`, так как возвращаемые значения будут отличаться друг от друга.

Неправильно
```bsl
ДатаПоследнегоОповещения = ТекущаяДатаСеанса();
ДатаСледующегоОповещения = РассчитатьДату() + ТекущаяДатаСеанса();
```

Правильно использовать ранее рассчитанные дату и время:
```bsl
ДатаПоследнегоОповещения = ТекущаяДатаСеанса();
ДатаСледующегоОповещения = РассчитатьДату() + ДатаПоследнегоОповещения;
```

связанные ишузы по методам ТекущаяДата, ТекущаяДатаСеанса

- https://github.com/1c-syntax/bsl-language-server/issues/924
- https://github.com/1c-syntax/bsl-language-server/issues/222

## Ссылка на источник, подтверждающее нарушение либо обоснование наличия проблемы

https://its.1c.ru/db/v8std/content/643/hdoc

### Тип [Статья на русском](https://1c-syntax.github.io/bsl-language-server/contributing/DiagnosticTypeAndSeverity)

* [x] :ant: Ошибка
* [ ] :cop: Уязвимость
* [ ] :guardsman: Потенциальная уязвимость
* [ ] :poop: Качество кода
* [ ] :trollface: Другое

### Важность [Статья на русском](https://1c-syntax.github.io/bsl-language-server/contributing/DiagnosticTypeAndSeverity)

* [ ] :broken_heart: Блокирующая / Blocker
* [ ] :heart: Критическая / Critical
* [x] :yellow_heart: Важная / Major
* [ ] :blue_heart: Незначительная / Minor
* [ ] :green_heart: Информационная / Info
* [ ] :revolving_hearts: Другое

### Тэги [Статья на русском](https://1c-syntax.github.io/bsl-language-server/contributing/DiagnosticTag)

* [x] `STANDARD` - "Нарушение стандартов 1С"
* [ ] `LOCKINOS` - "Не будет работать в другой ОС"
* [ ] `SQL` - "Проблема с запросом"
* [ ] `PERFORMANCE` - "Проблема производительности"
* [ ] `BRAINOVERLOAD` - "Непонятный код"
* [ ] `BADPRACTICE` - "Плохая практика программирования"
* [ ] `CLUMSY` - "Излишние действия"
* [ ] `DESIGN` - "Ошибка в проектировании"
* [x] `SUSPICIOUS` - "Подозрительный код"
* [x] `UNPREDICTABLE` - "Непредсказуемо работающий код"
* [ ] `DEPRECATED` - "Устаревшая функциональность"
* [ ] `ERROR` - "Ошибочная конструкция"
* [ ] `LOCALIZE` - "Проблемы локализации"

### Время на исправление (минут)

1 минута

**Comment by asosnoviy** [comment]
```bsl
НачалоЗамера = ТекущаяДатаСеанса();
ОкончаниеЗамера = ТекущаяДатаСеанса();
```

```bsl
НачалоЗамера = ТекущаяДатаСеанса();
ВремяВыполнения = ТекущаяДатаСеанса() - НачалоЗамера;
```

**Comment by artbear** [comment]
Да, в случае замеров нужно исключение, но подобных замеров все-таки не так много.
И в стандарте 1С об этом ни слова (

**Comment by zeegin** [comment]
> ```bsl
> НачалоЗамера = ТекущаяДатаСеанса();
> ОкончаниеЗамера = ТекущаяДатаСеанса();
> ```
> 
> ```bsl
> НачалоЗамера = ТекущаяДатаСеанса();
> ВремяВыполнения = ТекущаяДатаСеанса() - НачалоЗамера;
> ```

Пример знатный, но плохо когда в одном методе делается несколько замеров. Ведь замеряют обычно ключевые операции, а вызов нескольких ключевых операций в одном методе противоречит определению ключевой операции.

Либо это какие-то частные замеры, которые тоже стоит разделить на пару методов.
